### PR TITLE
fix: lint error, never used

### PR DESCRIPTION
**Pull request by yasudanaoya**
There was a lint error, so I fixed it.

### CODE REVIEW

File: plugins/gatsby-transformer-home-example-code/gatsby-node.js [hunk 1]
```diff
@@ -3,7 +3,6 @@
  */
 
 const crypto = require('crypto');
-const path = require('path');
```
Comment (yasudanaoya) [inline plugins/gatsby-transformer-home-example-code/gatsby-node.js:6]: An error was detected in another PR

https://github.com/reactjs/ja.reactjs.org/pull/456/files
image

**Comment by smikitky** [comment]
This fix belongs to the upstream repository (https://github.com/reactjs/reactjs.org), so please send a similar PR there. PRs merged into the upstream repo will eventually be merged into this Japanese fork. We would like to keep the discrepancy in the code as small as possible.

(But note that they are focusing on the [new beta docs](https://beta.reactjs.org/), so they may not merge non-urgent PRs for the current docs.)

**Comment by smikitky** [comment]
I just noticed the same fix has been already merged into the upstream, so you don't have to create the same PR.

https://github.com/reactjs/reactjs.org/pull/4115

**Comment by yasudanaoya** [comment]
thx

**Pull request closed without merging**